histogram.js

```diff
@@ -92,9 +92,10 @@ class HistogramElement extends HTMLElement {
       indicator.style.display = showSVG ? 'none' : 'block';
     }
     
+    document.addEventListener('toggle-loading-indicator-bamview', () => toggleSVGContainerAndIndicator.call(this, false));
```

Comment:
I think you're too focused on solving the immediate problem, and not thinking enough about the big picture. Think about how OICR is using these components. Does this event name make sense for them? What does the event represent?

Also, are you sure this is necessary for percents and histograms? Were they flashing too? If so, are you sure they need to listen to both `stats-stream-request` and the new event? Can they just use the new event?